### ISSUE TITLE
NTR: Cryptosporidium antigen present on immune assay or DFA of stool (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7919]
New term request Cryptosporidium antigen present on immune assay or DFA of stool (nadea3_190523010408): EIA; direct fluorescent antibody; fecal  UMLS: 2362654

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000838
- label: Positive fecal cryptosporidium antigen test
- parents: [Positive stool infectious agent test (HP:0034866)](https://hpo.jax.org/app/browse/term/HP:0034866)
- pmid: 
- def: Cryptosporidium antigen present in stool. Tests include direct fluorescent-antibody (DFA), which detects intact organisms, and enzyme immunoassays (EIAs), which detect soluble stool antigens.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7919
- synonyms:
